.github/workflows/upstream_testing.yml

```diff
@@ -10,4 +10,4 @@ jobs:
     uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@develop"
     with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
       invoke_context_name: "NAUTOBOT_CIRCUIT_MAINTENANCE"
-      plugin_name: "nautobot_circuit_maintenance"
+      plugin_name: "nautobot-app-circuit-maintenance"
```

Comment:
@snaselj please verify this does does not conflict with the upstream testing flow, more specifically if there's any state tracking being done based on the `plugin_name`

Reply:
Already verified, the plugin_name is used only for the local, not pushed, Docker image name:

https://github.com/nautobot/nautobot/blob/develop/.github/workflows/plugin_upstream_testing_base.yml#L42

Reply:
It could be possible replaced by lower cased `invoke_context_name` to get rid of `plugin_name` input.

Reply:
Let's get all of the initial Drift manager PRs merged, and then we can open a PR into core to update the testing CI then patch in the cookie and let DM quickly push it out.